### ISSUE TITLE
Decrease the app size

### ISSUE THREAD

**Issue by lucasbento** [joshghent/gifbar#5]
First of all, thank you for open-sourcing this, it's a really cool app!

I think there should be a small research on why the app size is `279.7 MB`, I believe it's possible to decrease that drastically as it's just a menubar app, I noticed that it's using packages like `lodash` which is pretty big but as it's using `parcel` (which I don't have much experience on) I'm not sure if it's doing properly tree-shaking or not.

**Comment by joshghent** [comment]
Yeah agreed, that is way too large. I'd more than welcome a PR if you are able to! Alternatively, I'll check into where I can cut the app size as soon as I can

**Comment by dhuang612** [comment]
Hi,

I can take a look at this

**Comment by joshghent** [comment]
All yours @dhuang612 - be great to have another contribution! 👍

**Comment by dhuang612** [comment]
created PR #112

**Comment by jamescallumyoung** [comment]
I wasn't involved in this when #112 was closed so I'll defer to you @joshghent ; can we consider this closed now?

**Comment by joshghent** [comment]
@jamescallumyoung Well let's check the app size now and see if it's acceptable. Maybe this can push forward the Github Actions CI work so we can measure the app size?

**Comment by jamescallumyoung** [comment]
Sounds good. Lets leave it for now then and see what the current size is.
